bitcoin/src/lib.rs

```diff
@@ -145,10 +145,11 @@ pub trait BitcoinCoreApi {
 
 pub struct LockedTransaction {
     pub transaction: Transaction,
-    _lock: OwnedMutexGuard<()>,
+    _lock: Option<OwnedMutexGuard<()>>,
```

Comment:
Why this change? I don't see None being used anywhere?

Reply:
https://github.com/interlay/polkabtc-clients/pull/55/files#diff-55b26c6a59093ae5631a640da179b541dc72b0da88ad229fcb19a1dfecacf1ddR620